lint.sh

```diff
@@ -3,10 +3,14 @@
 set -e #Exit entire script if any command fails
 
 # Download dependencies
+echo "Allowing plugin dependencies ..."
+composer global config --no-interaction allow-plugins.drupal/coder true
+composer global config --no-interaction allow-plugins.dealerdirect/phpcodesniffer-composer-installer true
+composer global config --no-interaction allow-plugins.sebastian/phpcpd true
```

Comment:
Not all of these are plugins... as in, pretty sure it's only `dealerdirect/phpcodesniffer-composer-installer` that has a plugin.

Reply:
Removed unnecessary lines